src/libaktualizr/storage/sqlstorage.cc

```diff
@@ -1158,28 +1152,27 @@ class SQLTargetWHandle : public StorageTargetWHandle {
     }
     written_size_ += size;
 
-    auto statement = db_.prepareStatement<int64_t, int64_t>("update target_images SET real_size = ? where rowid = ?;",
-                                                            static_cast<int64_t>(written_size_), row_id_);
-
-    int err = statement.step();
-    if (err != SQLITE_DONE) {
-      LOG_ERROR << "Could not save size in db: " << db_.errmsg();
-      throw exc;
-    }
-
-    wcommit();
-    db_.commitTransaction();
     return size;
   }
 
   void wcommit() override {
+    if (blob_ != nullptr) {
+      sqlite3_blob_close(blob_);
+      blob_ = nullptr;
+      auto statement = db_.prepareStatement<int64_t, int64_t>("update target_images SET real_size = ? where rowid = ?;",
```

Comment:
really small, but we upper case sql operations everywhere else, would be nice to keep a consistent style for easy grepping etc

Reply:
Sure, will fix.